### PR TITLE
Russian translation of "website-sections"

### PR DESCRIPTION
**Pull request by v1docq47**
Added by:
- outreach-docs/monero-outreach-docs/translations/ru/website-sections/

Translated:
- outreach-docs/monero-outreach-docs/translations/ru/website-sections/about_ru.md
- outreach-docs/monero-outreach-docs/translations/ru/website-sections/guerrila_toolkit_ru.md
- outreach-docs/monero-outreach-docs/translations/ru/website-sections/monero-typography_ru.md
- outreach-docs/monero-outreach-docs/translations/ru/website-sections/resources_ru.md
- outreach-docs/monero-outreach-docs/translations/ru/website-sections/stellumo-monero-fractal-art_ru.md

Calling to review:
@monerorus @LvMsterfild @TheFuzzStone

### CODE REVIEW

**Comment by v1docq47** [comment]
ping @erciccione ;)

File: monero-outreach-docs/translations/ru/website-sections/guerrila_toolkit_ru.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+4. Проецирование. При помощи обычного изображения, небольшого ноутбука и проектора вы можете добиться серьёзного воздействия. Как вариант можно использовать большой фонарик с наложенным на него трафаретом с логотипом Monero. Вспомните Бэтмена...
+5. Распространение информационных листовок. Массовая распечатка информации о Monero и распространение таких листовок на мероприятиях позволяет познакомить широкую (или новую) аудиторию с Monero.
+6. QR-коды. Массовая распечатка QR-кодов с небольшой рекламой, привлекающей внимание. Такие распечатки также можно распространять в местах проведения всевозможных мероприятий.
+7. Штемпелевание. Штемпелевание физических банкнот фиатных валют является эффективным и законным способом распространения информации о Monero. Согласно данным [stampmonero.com](https://stampmonero.com/), каждую долларовую банкноту, на которую вы поставите печать, могут увидеть 438 других человек.
+
+## Для дизайнеров
+
+1. [Monero Press Kit](https://www.getmonero.org/press-kit/) - здесь можно загрузить логотип Monero.
```
Comment (lh1008) [inline monero-outreach-docs/translations/ru/website-sections/guerrila_toolkit_ru.md]: We could add the Russian translation link for the getmonero.org site

https://www.getmonero.org/ru/press-kit/

[Monero Press Kit](https://www.getmonero.org/ru/press-kit/)

The same with the other links that go directly to the getmonero.org site.

File: monero-outreach-docs/translations/ru/website-sections/resources_ru.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
+# Ресурсы Monero
+
+У Monero самый широкий диапазон доступных ресурсов.
+
+Сообщество Monero велико и активно, в связи с чем к услугам как новичков, так и экспертов имеется множество руководств и открытых ресурсов. Это не полный список, а лишь краткий перечень проверенных источников, которые позволят познакомиться с Monero или продолжить уже начатое знакомство. Если у вас есть какие-либо предложения по этому разделу, пожалуйста, свяжитесь с нами!
+
+### С ЧЕГО НАЧАТЬ
+
+- [getmonero.org/downloads](https://ww.getmonero.org/downloads/)
```
Comment (lh1008) [inline monero-outreach-docs/translations/ru/website-sections/resources_ru.md]: https://www.getmonero.org/ru/press-kit/

[getmonero.org/downloads](https://www.getmonero.org/ru/press-kit/)

**Comment by lh1008** [comment]
Great work @v1docq47, thank you :).

File: monero-outreach-docs/translations/ru/website-sections/about_ru.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+## Обязательства Monero Outreach
+
+1. Способствовать распространению Monero и информации об этой монете, принимая участие во всевозможной, связанной с распространением взглядов, деятельности, в частности, путём организации связи с общественностью, проведения образовательных мероприятий и при помощи маркетинга.
+2. Уважать работу команды ведущих разработчиков и рабочих групп, входящих в сообщество, основополагающие принципы Monero, а также этические принципы Monero в любых ситуациях, не подвергая риску целостности того, что делает Monero выдающейся монетой.
+3. Сделать мир лучше за счёт распространения Monero. Нами движет вопрос: "Как мы можем подготовить наших детей к жизни в этом мире, если мы не подготовили сам мир для того, чтобы они жили в нём?"
+
+
+## Наша "дорожная карта"
+
+### БАЗОВЫЕ ЗАДАЧИ
```
[the remark this answers is not in the document]
Comment (v1docq47) [reply monero-outreach-docs/translations/ru/website-sections/about_ru.md]: Oh! Thank you! Indeed, not observed!

File: monero-outreach-docs/translations/ru/website-sections/guerrila_toolkit_ru.md (hunk 1 above)
Comment (v1docq47) [reply monero-outreach-docs/translations/ru/website-sections/guerrila_toolkit_ru.md]: Fixed

File: monero-outreach-docs/translations/ru/website-sections/resources_ru.md (hunk 2 above)
Comment (v1docq47) [reply monero-outreach-docs/translations/ru/website-sections/resources_ru.md]: Fixed

**Pull request merged**